src/python/RucioUtils.py

```diff
@@ -101,3 +101,28 @@ def getWritePFN(rucioClient=None, siteName='', lfn='',  # pylint: disable=danger
     logger.info(f"Will use {pfn} as stageout location")
 
     return pfn
+
+
+def getRuleQuota(rucioClient=None, ruleId=None):
+    """ return quota needed by this rule in Bytes """
+    size = 0
+    try:
+        rule = rucioClient.get_replication_rule(ruleId)
+    except RuleNotFound:
+        return 0
+    files = rucioClient.list_files(scope=rule['scope'], name= rule['name'])
+    size = sum(file['bytes'] for file in files)
+    return size
+
+
+def getTapeRecallUsage(rucioClient=None, account=None):
+    """ size of ongoing tape recalls for this account """
+    activity = 'Analysis Tape Recall'
+    rucioAccount = account
+    activity = 'Analysis TapeRecall'
+    rucioAccount = 'crab_tape_recall'
```

Comment:
```suggestion
```

Reply:
I fail to see what you suggest to change, can you explain ? suggested code looks identicall to original to me

Reply:
The suggestion is to delete L122-L123 because you already have it in L121-L122, and L123 make you always fetch the rule from `crab_tape_recall` account.

Reply:
**Thanks a lot** !!! Indeed the way it was, it would not work. It was a leftover of an initial version I used to test before Rahul made the needed changes to Rucio policiies. And I failed big time when "making it correct".